data/requirements.yml

```diff
@@ -5,14 +5,14 @@
+  body: "<p>While we recognize the validity of complete trust in the Lord for financial support, we need to have at least a general idea of how you plan to pay for your expenses during your stay in Ukraine. Each team member (including our full time staff) must be responsible for their own finances.</p><p>While participating in CMO, you will need to cover such expenses as rent, food, transportation, and whatever medical needs may arise. At this time, we estimate that you will need a <strong>minimum of $%{total_cost} per week</strong>. For more details on finances, please refer to the <a href=\"/downloads\">Info Pack</a>, page %{info_pack_finance_page}.</p>"
 
 - title: "Men Only"
   body: "<p>Women certainly play a vital role in evangelism world-wide. However, this particular project is not well-suited to a co-ed environment. One of the primary objectives of CMO is to provide missions training for men, and we believe that this is best accomplished without the distractions and complexities of a mixed-gender group.</p>"
 
-- title: "One-month Minimum"
-  body: "<p>We ask that all young men participating in CMO commit to at least one month in Ukraine. It is not practical to come for less. If possible, we strongly recommend that you plan to stay for the full seven-week duration of the project in order to receive maximum benefit from your training.</p><p>CMO %{current_project_year} will begin on %{project_start_date}. All team members are asked to arrive no later than %{max_arrival_date}. If you think you will have difficulty arranging your travel plans to concide with this window, please <a href=\"mailto:info@euroteamoutreach.org\">contact us</a> to discuss other options. More detailed travel information is available in the <a href=\"/downloads\">Info Pack</a>.</p><p>If you are a married man or a pastor (church leader), and feel that one month is more than you would be able to stay, please <a href=\"mailto:info@euroteamoutreach.org\">contact us</a> about this. We recognize that for some, a shorter stay would be more beneficial.</p>"
+- title: "Commitment of Service"
+  body: "<p>CMO %{current_project_year} will begin on %{project_start_date} and last through %{project_end_date}. We ask that all young men participating in CMO to come for the duration of the project. It is not practical to come for less.</p><p>All team members are expected to arrive on %{max_arrival_date}. If you think you will have difficulty arranging your travel plans to concide with this date, please <a href=\"mailto:info@euroteamoutreach.org\">contact us</a> to discuss other options. More detailed travel information is available in the <a href=\"/downloads\">Info Pack</a>.</p><p>If you are a married man or a church leader, and feel that you would not be able to stay for the whole project, please <a href=\"mailto:info@euroteamoutreach.org\">contact us</a> about this. We recognize that for some, a shorter stay is the only option.</p>"
```

Comment:
I think we have a problem with this sentence: _We ask that all young men participating in CMO to come for the duration of the project._

Need to remove word "to": _We ask that all young men participating in CMO ~to~ come for the duration of the project._